src/streamlink/plugins/nrk.py

```diff
@@ -13,7 +13,7 @@
 
 _id_re = re.compile(r"/(?:program|direkte|serie/[^/]+)/([^/]+)")
 _url_re = re.compile(r"https?://(tv|radio).nrk.no/")
-_api_baseurl_re = re.compile(r'apiBaseUrl:\s*"(?P<baseurl>[^"]+)"')
+_api_baseurl_re = re.compile(r'''apiBaseUrl:\s*["'](?P<baseurl>[^"']+)["']''')
```

Comment:
I like you use capture groups for the quotes then do a negative lookahead to continue until the next quote ;)
```python
r'''apiBaseUrl:\s*(['"])(?P<baseurl>(?:(?!\1).)*)\1'''
```

Probably overkill in this case though ;)